src/astro/databases/aws/redshift.py

```diff
@@ -0,0 +1,205 @@
+"""AWS Redshift table implementation."""
+from typing import Any, Dict, List, Optional, Tuple
+
+import pandas as pd
+from airflow.providers.amazon.aws.hooks.redshift_sql import RedshiftSQLHook
+
+from google.api_core.exceptions import (
```

Comment:
Is exceptions from google core api's relevant here?

Reply:
Perhaps you're reviewed from some stale commit. This is not the case with the latest commit. https://github.com/astronomer/astro-sdk/pull/639/files#diff-b6bfa1674a3103f5a361d30fed6d115dca45133e0ad89b8d956382d63da12daa